src/scripts/extensions/webExtensionBase/webExtensionWorker.ts

```diff
@@ -195,78 +194,80 @@ export class WebExtensionWorker extends ExtensionWorkerBase<W3CTab, number> {
 		return new Promise<boolean>((resolve, reject) => {
 			let popupWidth = 1000;
 			let popupHeight = 700;
-			let leftPosition: number = (screen && screen.width) ? Math.round((screen.width - popupWidth) / 2) : 0;
-			let topPosition: number = (screen && screen.height) ? Math.round((screen.height - popupHeight) / 2) : 0;
-
-			try {
-				/* As of 7/19/2016, Firefox does not yet supported the "focused" key for windows.create in the WebExtensions API */
-				/* See bug filed here: https://bugzilla.mozilla.org/show_bug.cgi?id=1213484 */
-				let windowOptions: chrome.windows.CreateData = {
-					height: popupHeight,
-					left: leftPosition,
-					top: topPosition,
-					type: "popup",
-					url: url,
-					width: popupWidth
-				};
-
-				if (this.clientInfo.get().clipperType !== ClientType.FirefoxExtension) {
-					windowOptions.focused = true;
-				}
-
-				WebExtension.browser.windows.create(windowOptions, (newWindow: Window) => {
-					let redirectOccurred = false;
-					let errorObject;
-					let correlationId: string;
+			WebExtension.browser.windows.getCurrent((currentWindow: Window) => {
+				let leftPosition: number = currentWindow.left + Math.round((currentWindow.width - popupWidth) / 2);
+				let topPosition: number = currentWindow.top + Math.round((currentWindow.height - popupHeight) / 2);
```

Comment:
Looks like there is a slight change in the position calculation here .. were we able to test if the placement is right in both edge and chrome?

Reply:
Yes, I have tested that the placement is right in Chrome (with both MV2 and MV3) as well as Edge (with MV2).